group_vars/model_tplink_cpe210_v1.yml

```diff
@@ -11,3 +11,4 @@ wireless_devices:
     htmode_prefix: HT
     path: platform/ahb/18100000.wmac
     ifname_hint: wlan2
+    antenna_gain: 9
```

Comment:
Why do we need this? Driver default works for me just fine:

![image](https://user-images.githubusercontent.com/198902/204219013-b63272fc-4961-460b-91df-48355b8054dc.png)